src/main/java/com/kamycz3q/coursesystemspringboot/personalData/personalDataObject/PersonalDataCompany.java

```diff
@@ -0,0 +1,23 @@
+package com.kamycz3q.coursesystemspringboot.personalData.personalDataObject;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+
+@Getter
+@Setter
+@Entity
+public class PersonalDataCompany {
+    @Id
+    @GeneratedValue(
+            strategy = GenerationType.AUTO
+    )
+    private Long id;
+    private String name, surname, email, companyName;
+    private String city, address, postCode;
+    private Long nip;
```

Comment:
To samo co z nrem PESEL. Btw PESEL jako liczba będzie zawsze większy od NIPu, a to przy NIP zastosowałeś typ Long. Nie jestem pewien czy nr PESEL nie wykracza już poza skalę typu Integer.